src/libs/KeyboardShortcut/index.js

```diff
@@ -145,9 +150,10 @@ function getPlatformEquivalentForKeys(keys) {
  * @param {Boolean|Function} [shouldBubble] Should the event bubble?
  * @param {Number} [priority] The position the callback should take in the stack. 0 means top priority, and 1 means less priority than the most recently added.
  * @param {Boolean} [shouldPreventDefault] Should call event.preventDefault after callback?
+ * @param {Array<String>} [excludeNodes] Do not capture key events targeting excluded nodes (i.e. do not prevent default and let the event bubble)
```

Comment:
```suggestion
 * @param {Array<String>} [excludedNodes] Do not capture key events targeting excluded nodes (i.e. do not prevent default and let the event bubble)
```

Reply:
I had the same idea for that change 😅. Thanks for the suggestion. Looks better now!